examples/axisartist/demo_floating_axis.py

```diff
@@ -1,30 +1,35 @@
 """
-An experimental support for curvilinear grid.
+Axis Within Rectangular Frame
```

Comment:
First word sounds better.
Sorry @sirlittle : do you mind quickly updating this?